app/Nova/Actions/ExportUsersBuzzCardAccess.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova\Actions;
+
+use App\Models\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Facades\URL;
+use Laravel\Nova\Actions\Action;
+use Laravel\Nova\Fields\ActionFields;
+use Laravel\Nova\Fields\Select;
+
+class ExportUsersBuzzCardAccess extends Action
+{
+    use InteractsWithQueue, Queueable;
+
+    /**
+     * The displayable name of the action.
+     *
+     * @var string
+     */
+    public $name = 'Export BuzzCard Access List';
+
+    /**
+     * Disables action log events for this action.
+     *
+     * @var bool
+     */
+    public $withoutActionEvents = true;
+
+    /**
+     * Perform the action on the given models.
+     *
+     * @param  \Laravel\Nova\Fields\ActionFields  $fields
+     * @param  \Illuminate\Support\Collection  $models
+     * @return mixed
+     */
+    public function handle(ActionFields $fields, Collection $models)
+    {
+        $population = $fields->population;
+        if ($population === 'core') {
+            $users = User::select('gtid')->BuzzCardAccessEligible()
```

Comment:
For both queries, also exclude

- Users whose primary affiliation is not `student`, `faculty`, or `staff`
- Users with no attendance in the last 6(?) months (maybe skip until people counter is feeding into apiary)
- Users who have ever paid for a package that is not restricted to students

Reply:
I don't agree with removing people with no attendance, especially this year. There's lots of people who don't go to the shop regularly and given these exports are only run semesterly that could be a real pain for some members. The other exclusions are good.